packages/dialect-react-ui/components/Chat/screens/Main/pages/CreateThreadPage/CreateThread.tsx

```diff
@@ -248,11 +291,19 @@ export default function CreateThread({
     const delayDebounceFn = setTimeout(async () => {
       if (address.length === 0) {
         setValidAddress(false);
-      } else if (address.charAt(0) != '@') {
+      } else if (address.slice(address.length - 4) === '.sol') {
```

Comment:
I'm assuming `- 4` is because `.sol` has 4 symbols.
Minor thing, but wouldn't be `.match` better?
```
const BONFIDA_DOMAIN_EXT = '.sol';
if (address.match(BONFIDA_DOMAIN_EXT)) { ... }
```

Reply:
Good call, let's go with the const `.sol` and call `.match`